openmrs/omod/src/main/java/org/projectbuendia/openmrs/web/controller/ProfileManager.java

```diff
@@ -130,7 +160,7 @@ String getNextVersionedFilename(String name) {
             } else if (n.startsWith(prefix) && n.endsWith(ext)) {
                 try {
                     version = Integer.parseInt(
-                        n.substring(prefix.length(), n.length() - ext.length()));
+                            n.substring(prefix.length(), n.length() - ext.length()));
```

Comment:
I think IntelliJ will get it right if you use the project settings file here: https://github.com/projectbuendia/buendia/blob/dev/intellij-idea-settings.jar

Instructions on how to import it are here: https://github.com/projectbuendia/buendia/wiki/Java-Style

Could you give it a try and see if it fixes the indentation to match the style guide?